docker/docker/Dockerfile.linux.amd64

```diff
@@ -5,12 +5,20 @@ ENV DOCKER_HOST=unix:///var/run/docker.sock
 ENV DOCKER_BUILDKIT=1
 ENV BUILDKIT_PROGRESS=plain
 ENV DOCKER_CLI_EXPERIMENTAL=enabled
+ENV PLUGIN_BUILDKIT_ASSETS_DIR=/buildkit
 
 ARG BUILDX_URL=https://github.com/docker/buildx/releases/download/v0.18.0/buildx-v0.18.0.linux-amd64
 
 RUN mkdir -p $HOME/.docker/cli-plugins && \
     wget -O $HOME/.docker/cli-plugins/docker-buildx $BUILDX_URL && \
     chmod a+x $HOME/.docker/cli-plugins/docker-buildx
 
+# Create /buildkit directory
+RUN mkdir -p /buildkit
+
+# Add buildkit.tar and version
+COPY buildkit/buildkit.tar /buildkit/buildkit.tar
+COPY buildkit/version.json /buildkit/version.json
```

Comment:
Let's keep this version file out of the code. It should be only used during the build. We can then do an environment variable inside the docker env. (which can be overridden if we don't like the kit)